snuba/query/processors/uuid_column_processor.py

```diff
@@ -26,6 +27,141 @@
 metrics = MetricsWrapper(environment.metrics, "api.query.uuid_processor")
 
 
+class UUIDTranslationError(Exception):
+    pass
```

Comment:
Could you please extend `ValidationException` ?
That will return an HTTP 400 instead of a 500. 
https://github.com/getsentry/snuba/blob/16109210ba80131ada67d42eb706a3046eefbd26/snuba/query/exceptions.py#L14

If the queyr is invalid it should not count against the snuba failure rate.

Reply:
This is going to return a 200 not 500 (or 400). See the rationale here: https://github.com/getsentry/snuba/pull/1754#discussion_r593407751

Let me know if you agree